portal/index.php

```diff
@@ -526,39 +537,40 @@ function enableVerifyBtn(){
             </form>
         <?php } else {
             ?> <!-- Main logon -->
-        <img class="img-fluid login-logo" src='<?php echo $logoSrc; ?>'>
-        <form class="text-center" action="get_patient_info.php" method="POST" onsubmit="return process()">
-            <?php if (isset($redirectUrl)) { ?>
-                <input id="redirect" type="hidden" name="redirect" value="<?php echo attr($redirectUrl); ?>" />
-            <?php } ?>
-            <fieldset>
-                <legend class="bg-primary text-white pt-2 py-1"><h3><?php echo xlt('Patient Portal Login'); ?></h3></legend>
-                <div class="jumbotron jumbotron-fluid px-5 py-3">
-                    <div class="form-row my-3">
+        <img class="img-fluid w-25" src='<?php echo $logoSrc; ?>'>
+        <div class="form-container sign-in-container w-50">
+            <form class="patient-form" action="get_patient_info.php" method="POST" onsubmit="return process()" style="padding-left: 250px">
```

Comment:
How does padding in a style add to responsiveness?
If anything we should remove the old portal theme, implement openemr dark or light theme selectable from Config and  include BS in header.

Reply:
That was a desperate effort to get the screen to behave how I wanted it to. It was the only way I could get the form in the center of the screen. 
As I said, this is the first draft so you could see where I was headed.